### DISCUSSION TITLE
Site build fails with recent mkdocs-material version (9.5.45)

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#7724] [Q&A] [answered]
The CI I use to build and publish my site with - including a blog using the blog plugin - started to fail with the 9.5.45 release of Material for MkDocs.
My guess here is, that the Fix for #7708 unintentionally caused issues on sites who worked fine so far.

The error seems to be `TypeError: can't compare offset-naive and offset-aware datetimes` and the CI logs can be found here: https://ci.codeberg.org/repos/12442/pipeline/203/4

An example blog post can be found here: https://codeberg.org/Andre601/website/src/branch/main/docs/blog/posts/2024/11-14-add-fediverse-creator-tag.md
(Note, page loading may be a bit slow)

I make a discussion rather than an issue, as I'm unsure on wether I can create a reproducable example for this issue.

**Comment by kamilkrzyskow** [comment, marked as answer]
Thanks for reporting, tracked in
- https://github.com/squidfunk/mkdocs-material/issues/7725
